### PR TITLE
Fix format="text" in all tools.

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

**Comment by davebx** [comment]
:+1:

File: tool_collections/gatk/print_reads/print_reads.xml [hunk 1]
```diff
@@ -86,15 +86,6 @@
     <data format="bam" name="output_bam" label="${tool.name} on ${on_string} (BAM)" />
     <data format="txt" name="output_log" label="${tool.name} on ${on_string} (log)" />
   </outputs>
-    <param name="number" type="integer" value="-1" label="Print the first n reads from the file, discarding the rest" />
```
Comment (martenson) [inline tool_collections/gatk/print_reads/print_reads.xml]: what did this do here? :D

Comment (davebx) [inline tool_collections/gatk/print_reads/print_reads.xml]: This ... makes no sense.

Comment (nsoranzo) [inline tool_collections/gatk/print_reads/print_reads.xml]: Somehow-duplicated code.

**Pull request merged**